udata/core/dataset/factories.py

```diff
@@ -58,3 +59,8 @@ class Meta:
 
     id = factory.Sequence(lambda n: '{0}-{1}'.format(faker.word(), n))
     title = factory.LazyAttribute(lambda o: faker.sentence())
+
+
+class UserCreatedDatasetFactory(MongoEngineFactory):
```

Comment:
In this case (empty factories only depending on real parameters), I prefer that we use the model directly:
- do not hide anything (less complexity when not needed)
- do not add code

I you want to avoid constructor+save(), just use `Model.objects.create()`, it will do the same thin in one call